package.json

```diff
@@ -298,10 +298,16 @@
                 "type": "object",
                 "description": "When present, this tells the debugger to connect to a remote computer using another executable as a pipe that will relay standard input/output between VS Code and the .NET Core debugger backend executable (clrdbg).",
                 "default": {
+                  "pipeCwd": "${workspaceRoot}",
                   "pipeProgram": "enter the fully qualified path for the pipe program name, for example 'c:\\tools\\plink.exe'",
                   "pipeArgs": []
                 },
                 "properties" : {
+                  "pipeCwd": {
+                    "type": "string",
+                    "description": "The fully qualified path where the pipe command to execute.",
```

Comment:
The description sentence sounds grammatically incorrect. Maybe something like "The fully qualified path to the working directory for the pipe program"?